### ISSUE TITLE
WCAG 2.2 本体 : 用語「コントラスト比 (contrast ratio)」の注記 4

### ISSUE THREAD

**Issue by caztcha** [waic/wcag22#2188]
原文
> Background color is the specified color of content over which the text is to be rendered in normal usage. It is a failure if no background color is specified when the text color is specified, because the user's default background color is unknown and cannot be evaluated for sufficient contrast. For the same reason, it is a failure if no text color is specified when a background color is specified.

現行訳
> 背景色は、テキストが通常の使用においてレンダリングされるときに背景となるコンテンツの色として指定された色である。テキストの色は指定されているが背景色が指定されていない場合、利用者のデフォルトの背景色は未知であり、コントラストが十分かどうかを評価することができないため、失敗となる。同じ理由で、背景色が指定されているがテキストの色が指定されていない場合も失敗となる。

修正案 (rendered in normal nusage の訳を、注記 3 の修正 #2187 と合わせる)
> 背景色は、テキストが通常の用法でレンダリングされるときに背景となるコンテンツの色として指定された色である。テキストの色は指定されているが背景色が指定されていない場合、利用者のデフォルトの背景色は未知であり、コントラストが十分かどうかを評価することができないため、失敗となる。同じ理由で、背景色が指定されているがテキストの色が指定されていない場合も失敗となる。
